utilities/python-helpers/test_python_helpers.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from .listMethods import exprToList, listToExpr
+
+
+class TestListMethods(unittest.TestCase):
+    
+    def test_listToExpr(self):
+        expected_result_1 = "(A B C)"
+        received_result_1 = listToExpr("(Cons A (Cons B (Cons C Nil)))")
+        expected_result_2 = "((mkInst (0 1 2)) (B) (C) D E)"
```

Comment:
This shouldn't have been the expected result if there are other datastructures inside. The function should only work on the top level list without changing the structure of any thing else that is an element of the list.

Reply:
Yeah, that's what I meant. It should only work top level and not change any other nested datastructure.